migrations/dimensions-config/bf6308af80aa.py

```diff
@@ -0,0 +1,402 @@
+"""Migration script for dimensions.yaml namespace=daf_butler version=2.
+
+Revision ID: bf6308af80aa
+Revises: 380002bcbb26
+Create Date: 2022-05-16 12:11:17.906600
+
+"""
+import logging
+from typing import Dict, List, Optional
+
+import sqlalchemy as sa
+from alembic import context, op
+from lsst.daf.butler_migrate.butler_attributes import ButlerAttributes
+from lsst.daf.butler_migrate.digests import get_digest
+from lsst.daf.butler_migrate.shrink import shrinkDatabaseEntityName
+
+# revision identifiers, used by Alembic.
+revision = "bf6308af80aa"
+down_revision = "380002bcbb26"
+branch_labels = None
+depends_on = None
+
+_LOG = logging.getLogger(__name__)
+
+
+def upgrade() -> None:
+    """Upgrade from version 1 to version 2.
+
+    Summary of changes:
+
+        - new table `visit_system_membership`
+        - new columns in dimension tables:
+            - instrument table adds visit_system column
+            - exposure table adds seq_start, seq_end, has_simulated, azimuth
+            - visit table adds seq_num and azimuth
+        - some of these new columns need to be populated with non-NULL values
+    """
+
+    config = context.config
+    has_simulated_str = config.get_section_option("daf_butler_migrate_options", "has_simulated")
+    if has_simulated_str is None:
+        raise ValueError(
+            "This migration script requires `has_simulated` option, please use "
```

Comment:
Maybe: "This migration script requires a default value be provided for the `has_simulated` field in the `exposure` record"